test/testvm.py

```diff
@@ -714,6 +709,12 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
         drive_to_start = self._image_root
         if os.path.exists(self._image_image):
             drive_to_start = self._image_image
+        elif os.path.exists(self._image_original):
+            subprocess.check_call([ "qemu-img", "create", "-q",
+                                    "-f", "qcow2",
+                                    "-o", "backing_file=%s" % self._image_original,
```

Comment:
We should probably do a follow up that checks that cockpit isn't checked out to a path with a space or comma in the name. Otherwise stuff like this will break.